ci-operator/populate-secrets-from-bitwarden.sh

```diff
@@ -16,6 +15,29 @@ set -o xtrace
 #          content of the BitWarden vault after your current session was started,
 #          you will need to create a new session to be able to view those changes.
 
+CURRENT_OC_CONTEXT=$(oc config current-context)
+readonly CURRENT_OC_CONTEXT
+OC_PROJECT=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f1)
+readonly OC_PROJECT
+OC_CLUSTER=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f2)
+readonly OC_CLUSTER
+OC_USER=$(echo ${CURRENT_OC_CONTEXT} | cut -d "/" -f3)
+readonly OC_USER
+
+if [[ "${OC_CLUSTER}" != "api-ci-openshift-org:443" ]]; then
+	>&2 echo "current cluster ${OC_CLUSTER} is not our ci-cluster ... please run 'oc login https://api.ci.openshift.org' first!"
```

Comment:
@stevekuznetsov will love this.

Reply:
I think this will need `"${OC_CLUSTER}" != *"api-ci-openshift-org:443"`

Reply:
What is the value in your context?

Reply:
You are right, `api-ci-openshift-org:443`.